app/fissile.go

```diff
@@ -452,8 +461,57 @@ func (f *Fissile) GeneratePackagesRoleImage(repository string, roleManifest *mod
 	return nil
 }
 
+// GeneratePackagesRoleTarball builds a tarball snapshot of the build context
+// for the docker image for the packages layer where all packages are included
+func (f *Fissile) GeneratePackagesRoleTarball(repository string, roleManifest *model.RoleManifest, noBuild, force bool, roles model.Roles, outputDirectory string, packagesImageBuilder *builder.PackagesImageBuilder) error {
+	if len(f.releases) == 0 {
+		return fmt.Errorf("Releases not loaded")
+	}
+
+	packagesLayerImageName, err := packagesImageBuilder.GetRolePackageImageName(roleManifest, roles)
+	if err != nil {
+		return fmt.Errorf("Error finding role's package name: %s", err.Error())
```

Comment:
Thought you didn't need `err.Error()` for `%s` either. (It was here because it was copied old code; will fix, though)